aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/Orchestrator.java

```diff
@@ -51,7 +51,7 @@
  */
 public final class Orchestrator {
     private static final Logger LOG = Amplify.Logging.forNamespace("amplify:aws-datastore");
-    private static final long LOCAL_OP_TIMEOUT_SECONDS = 5;
+    private static final long LOCAL_OP_TIMEOUT_SECONDS = 7;
```

Comment:
The semaphore queue is used here to manage concurrent start and stop of the orchestrator. The tests are timing out because orchestrator can't start in the specified time, I have see[n this to be an issue with ](https://github.com/aws-amplify/amplify-android/issues/1403)customers as well. That's why increasing the time out.